apis/shared-dimensions/lib/domain/shared-dimensions.ts

```diff
@@ -71,3 +71,35 @@ export function getSharedTerms({ sharedDimension, freetextQuery, validThrough, l
       ${term} ?p ?o .
     `
 }
+
+function textSearch(subject: Term, predicate: Term, textQuery: string): TemplateResult<SparqlExecuteOptions> {
```

Comment:
You don't really need the return type here.

And it could be `{ SparqlTemplateResult } from '@tpluscode/string-builder'`